docs/docs-content/clusters/edge/trusted-boot/edgeforge/build-trusted-iso.md

```diff
@@ -103,6 +111,15 @@ This document guides you through the process of producing Edge Installer ISOs th
 
    :::warning
 
+   Adding software dependencies in the Dockerfile will cause the size of the Extensible Firmware Interface (EFI) file to
+   grow. Most hardware has a limit on the size of the EFI that it can boot. Make sure you do not include too many
+   dependencies can cause the EFI file to grow larger than the boot limit. For more information, refer to
+   [Check EFI Boot Limit](./check-efi-limit.md).
```

Comment:
```suggestion
   grow. Most hardware has a limit on the size of the EFI that it can boot. Make sure you do not include too many
   dependencies that can cause the EFI file to grow larger than the boot limit. For more information, refer to
   [Check EFI Boot Limit](./check-efi-limit.md).
```